01-shapefile-attributes.Rmd

```diff
@@ -235,15 +221,14 @@ lines_HARV$TYPE
 
 ```
 
+Notice that two of our attributes have space in them: stone wall and woods road.
```

Comment:
Space? what does that mean? Perhaps a missing value? 

Also do you mean two of our attributes (i.e. TYPE is an attribute) Type has 6 (or perhaps 5 or 4 without viewing the data) unique attribute VALUES. 

Maybe show me this too because i see 4 "levels" or UNIQUE VALUES for the type attribute in the online version